lib/hooks/core/hook-http.js

```diff
@@ -45,7 +45,7 @@ function requestWrap(request) {
     }
 
     if (!cls.getRootContext()) {
-      agent.logger_.warn('Cannot create http span outside of a supported framework.');
+      agent.logger.warn('Cannot create http span outside of a supported framework.');
```

Comment:
While you're here, can you change the wording to 'Cannot trace outbound http requests outside of a supported http framework', or some variation of that.